dom-manipulation/main.js

```diff
@@ -0,0 +1,25 @@
+function handleClick(event) {
+  var $clickcount = document.querySelector('.click-count');
+  $clickcount.textContent = 'Clicks: ' + incrementer++;
+}
+
+function changeText(event) {
+  if (incrementer <= 4) {
+    $button.className = 'hot-button cold';
+  } else if (incrementer < 7) {
+    $button.className = 'hot-button cool';
+  } else if (incrementer < 10) {
+    $button.className = 'hot-button tepid';
+  } else if (incrementer < 13) {
+    $button.className = 'hot-button warm';
+  } else if (incrementer < 16) {
+    $button.className = 'hot-button hot';
+  } else if (incrementer > 17) {
+    $button.className = 'hot-button nuclear';
+  }
+}
+
+var $button = document.querySelector('.hot-button');
+var incrementer = 0;
+$button.addEventListener('click', handleClick);
+$button.addEventListener('click', changeText);
```

Comment:
Instead of having two separate event handlers for the `click` event on this one element, you should do all of this work in one function, especially because the work is related to one another.